tests/test_ls.py

```diff
@@ -14,9 +14,8 @@ def test_obj():
     o.foo.bar.something = Object()
     o.foo.bar.aaa = Object()
     o.foo.bar.bbb = Object()
-    o.foo.bar._something_else = lambda: None
+    o.foo.bar._something_else = dict  # a callable (lambda recurses infinitely in Python 2.7 when depth=None)
```

Comment:
@yaJShah can you please write a test with a lambda and fix the recursion problem in a separate issue? Thanks.